CUSTOMIZING.md

```diff
@@ -587,4 +587,19 @@ Die Offline Synchronisation ist ähnlich wie der Onlinemodus erweiterbar. Für d
 
 Die Anzeige von Listen und Dokumenten nutzt auch 'listStlyes' und 'documentStyles'. Für die synchronisierten Firmen und Kontakte gibt es in der JavaScript-Bibliothek 'app_getConfig' jeweils einen 'listStyle' und 'documentStyle' mit dem Prefix "OFFLINE". Diese können für Anpassungen über die JavaScript-Bibliothek 'app_customization' verändert werden.
 
+# Tablet mode
+
+Mit goMobile Pro Version 5.0 wurde der sogenannte Tablet mode eingeführt. Bei Anpassungen kann man mit dem Property `size` steuern wie breit ein Component auf einem Tablet angezeigt wird. Der Tablet mode basiert auf einem Grid, dass mit 12 Spalten aufgebaut ist. Der Wert von `size` ist Anzahl von Spalten die ein Component auf einem Tablet einnehmen soll. Wenn man beispielsweise eine zweispaltige Darstellung wünscht, dann müssen alle Components ein `size: '6'` bekommen. Einzelne Components können aber auch breiter oder schmaler gestaltet werden und das Grid richtet sich dementsprechend aus.
```

Comment:
Schreibt man im Deutschen dann 'mode' wirklich klein?
Weiterhin würde in ein paar Sätze wahrscheinlich ein wenige drehen und minimal kürzen.
Ab goMobile Pro Version 5.0 gibt es den sogenannte Tablet mode. Mit dem zugehörigen Property 'size' kann die Breite von Component bei der Anzeige auf einem Tablet gesteuert werden. Der Tablet-Mode basiert auf einem 12-spaltigen Grid und der Wert von 'size' steuert dabei die Anzahl der Spalten, welche von einem Component verwendet werden sollen. Wenn beispielsweise eine zweispaltige Darstellung gewünscht ist, muss bei allen Components der Wert '6' ins Property 'size' geschrieben werden. Einzelne Components...

Reply:
Hab ich so eingefügt.